core/templates/dev/head/collection_editor/CollectionEditor.js

```diff
@@ -28,6 +28,64 @@ oppia.constant(
 oppia.constant(
   'EXPLORATION_SUMMARY_DATA_URL_TEMPLATE', '/explorationsummarieshandler/data');
 
+// Sorts the node list for display in collection editor frontend. The list
+// starts with a startNode which contains no prerequisite skills. After that
+// nodes are added to the list if their prerequisite skills match the acquired
+// skills of the last node that was added to the node list.
+oppia.filter('customSorter', function() {
```

Comment:
This probably isn't the name we want to have for this. This is trying to sort collection nodes by their progression order. Maybe it could be named ``sortCollectionNodesByProgressionOrder``.

Also, this filter should go in ``filters.js``